schemas/resources-response.yml

```diff
@@ -0,0 +1,52 @@
+---
+  title: "WorkerType Resources Response"
+  type: object
+  properties:
+    resources:
+      type: array
+      description: |
+        List of objects representing AWS resources associated with this worker type.
+      items:
+        type: object
+        properties:
+          instanceId:
+            type: string
+            description: EC2 Instance ID (if an instance exists)
+          spotRequestId:
+            type: string
+            description: EC2 Spot Request ID (if the instance is a spot instance)
+          amiId:
+            type: string
+            description: Amazon Machine Image ID
+          type:
+            type: string
+            description: EC2 instance type
+          region:
+            type: string
+            description: AWS Region
+          zone:
+            type: string
+            description: AWS Availability Zone
+          state:
+            type: string
+            description: |
+              Resource state; "running" indicates an instance able to execute tasks.
```

Comment:
Sure, but I have no idea what the acceptable values are -- and missing one
will cause 500 errors when an instance is in that state..
​